### PR TITLE
[Security] Require nokogiri 1.10.4 or greater

### LINKED ISSUE #636: [Security] Bump nokogiri from 1.10.2 to 1.10.3
*Sourced from [The Ruby Advisory Database](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/nokogiri/CVE-2019-11068.yml).*

> **Nokogiri gem, via libxslt, is affected by improper access control vulnerability**
> Nokogiri v1.10.3 has been released.
> 
> This is a security release. It addresses a CVE in upstream libxslt rated as
> "Priority: medium" by Canonical, and "NVD Severity: high" by Debian. More
> details are available below.
> 
> If you're using your distro's system libraries, rather than Nokogiri's
> vendored libraries, there's no security need to upgrade at this time, though
> you may want to check with your distro whether they've patched this
> (Canonical has patched Ubuntu packages). Note that this patch is not yet (as
> of 2019-04-22) in an upstream release of libxslt.
> 
> Full details about the security update are available in Github Issue
> [#1892] https://github-redirect.dependabot.com/sparklemotion/nokogiri/issues/1892.
> 
> ---
> 
> CVE-2019-11068

### PR DESCRIPTION
**Pull request by cetinajero**
Addresses CVE-2019-11068
https://nvd.nist.gov/vuln/detail/CVE-2019-11068

**Nokogiri gem, via libxslt, is affected by improper access control vulnerability**

Fixes #636

### CODE REVIEW

**Comment by RDIL** [comment]
Can you add `fixes #636` to the description please

**Comment by cetinajero** [comment]
Sure @RDIL!

**Comment by RDIL** [comment]
Thank you!

File: github-pages.gemspec [hunk 1]
```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency("mercenary", "~> 0.3")
   s.add_dependency("terminal-table", "~> 1.4")
-  s.add_dependency("nokogiri", ">= 1.8.5", "< 2.0")
+  s.add_dependency("nokogiri", ">= 1.10.3", "< 2.0")
```
Comment (parkr) [inline github-pages.gemspec]: I believe this should be 1.19.4 now

Comment (cetinajero) [reply github-pages.gemspec]: Updated 👋🏼

**Comment by foucdeg** [comment]
@parkr is this going to be in a release soon? I'm trying to fix my Github security warnings.

**Comment by parkr** [comment]
You should be able to add the following to your `Gemfile` to fix up warnings:

```ruby
gem "nokogiri", ">= 1.10.4"
```

We'll try to get a release out soon!

**Comment by dlion** [comment]
Any update about the release? I'm still receiving warnings about it :)

**Comment by cetinajero** [comment]
You should take a look to [v199](https://github.com/github/pages-gem/releases/tag/v199) @dlion.

That version was released 4 days ago and includes this PR.

**Pull request merged**